Content.Shared/_RMC14/Xenonids/Construction/Tunnel/XenoTunnelSpawnerComponent.cs

```diff
@@ -0,0 +1,21 @@
+using Robust.Shared.Prototypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared._RMC14.Xenonids.Construction.Tunnel;
+
+/// <summary>
+/// For mapping purposes, the gamerule should use the information in this component to spawn tunnels with the proper name and hive
+/// </summary>
+[RegisterComponent]
+public sealed partial class XenoTunnelSpawnerComponent : Component
+{
+    [DataField]
+    public string? TunnelName;
+
+    [DataField]
+    public string? HiveName;
```

Comment:
This doesn't look used from the code, distress signal is the system creating the hive round-start anyways
```suggestion
```

Reply:
It's supposed to be like, future proofed if there's multiple hives.
Though it's not very useful if they have randomly generated hive names.